### PR TITLE
Fix paging

### PR DESCRIPTION
**Pull request by anhdh-3069**
## Related Tickets
- [#49970](https://edu-redmine.sun-asterisk.vn/issues/49970)

## WHAT (optional)
- Sửa lại hiển thị phân trang đúng khi search 

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by anhdh-3069** [comment]
ready

File: app/views/admin/users/_users.html.erb [hunk 1]
```diff
@@ -12,3 +12,6 @@
     <%= render @users %>
   </tbody>
 </table>
+<div class="align-center">
+    <%== pagy_bootstrap_nav @pagy %>
```
Comment (ngocvt-0484) [inline app/views/admin/users/_users.html.erb:16]: sai indent

**Comment by anhdh-3069** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**